core/src/main/scala/akka/persistence/jdbc/state/scaladsl/package.scala

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2014 - 2019 Dennis Vriend <https://github.com/dnvriend>
+ * Copyright (C) 2019 - 2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.jdbc.state
+
+import akka.persistence.query._
+package object scaladsl {
```

Comment:
Package objects going away for Scala 3 so better to not introduce new ones.

Reply:
done ..